Makefile

```diff
@@ -34,7 +34,7 @@ run: build
 	docker run -e MYHOSTNAME=localhost -e PLAIN_AUTH $(IMAGE_REPOSITORY)
 
 test: build
-	MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} make -C tests test
+	URL="docker=${IMAGE_REPOSITORY}" make -C tests test
```

Comment:
Why did you delete MODULE=docker? I guess, it is prerequisite, or MODULE=docker is default?

Reply:
no, it is connected with second change in `config.yaml` `default_module: docker`